.github/workflows/gradle.yml

```diff
@@ -20,6 +20,10 @@ jobs:
 
       - uses: actions/checkout@v3
 
+      - name: Setup Protobuf
+        run: |
+          sudo apt-get update -y
+          sudo apt-get install -y protobuf-compiler
```

Comment:
```suggestion
```

We don't need this step. Ask @OlehSarapuk

Reply:
Your right
There are all needed dependencies and plugin to do it automatically

Reply:
@ajax-gnatiuk-s 
Ur right, I'll rework this